cmd/kueuectl/app/create/create_clusterqueue.go

```diff
@@ -136,11 +136,17 @@ func NewClusterQueueCmd(clientGetter util.ClientGetter, streams genericiooptions
 		Long:                  cqLong,
 		Example:               cqExample,
 		Args:                  cobra.MatchAll(cobra.ExactArgs(1), cobra.OnlyValidArgs),
-		Run: func(cmd *cobra.Command, args []string) {
-			ctx := cmd.Context()
-			cobra.CheckErr(o.Complete(clientGetter, cmd, args))
-			cobra.CheckErr(o.validate())
-			cobra.CheckErr(o.Run(ctx))
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
```

Comment:
```suggestion
		SilenceUsage: true
		RunE: func(cmd *cobra.Command, args []string) error {
```

Could we set the `SilenceUsage` option in the struct instead of insertion?

Reply:
No, because on this case we will silence `--help` printing.

Reply:
By default if we will set not args or invalid args cobra print help. If we will add it to struct before running it will not print.

Reply:
I see. Thanks.
/lgtm